libcontainer/intelrdt/intelrdt.go

```diff
@@ -548,26 +557,45 @@ func GetIntelRdtPath(id string) (string, error) {
 	return path, nil
 }
 
+func GetMonGroupIntelRdtPath(id string) (string, error) {
+	rootPath, err := getIntelRdtRoot()
+	if err != nil {
+		return "", err
+	}
+
+	path := filepath.Join(rootPath, MonGroupDirectoryName, id)
+	return path, nil
+}
+
 // Applies Intel RDT configuration to the process with the specified pid
 func (m *IntelRdtManager) Apply(pid int) (err error) {
-	// If intelRdt is not specified in config, we do nothing
-	if m.Config.IntelRdt == nil {
+	switch m.Type {
+	case CTRL_MON:
+		// If intelRdt is not specified in config, we do nothing
+		if m.Config.IntelRdt == nil {
+			return nil
+		}
+		d, err := getIntelRdtData(m.Config, pid)
```

Comment:
Could change `d` variable to variable with more meaningful name e.g. `rdtData` or `data`? imho it'll improve code readability.